### PR TITLE
Practice prompt calibration

### PR DESCRIPTION
**Pull request by wpietri**

### CODE REVIEW

File: src/modelbench/run.py [hunk 1]
```diff
@@ -269,23 +269,25 @@ def calibrate(update: bool, file) -> None:
...
         print("Can't load private annotators needed for calibration")
         exit(1)
 
-    benchmarks = [GeneralPurposeAiChatBenchmark()]
-    for l in Locale:
+    benchmarks = []
+    for l in [Locale.EN_US]:
         benchmarks.append(GeneralPurposeAiChatBenchmarkV1(l, "ensemble"))
-    run_result = run_benchmarks_for_suts(benchmarks, reference_suts, 100)
+    run_result = run_benchmarks_for_suts(benchmarks, reference_suts, 2000)
```
Comment (bkorycki) [inline src/modelbench/run.py:282]: I'm curious about the reasoning behind 2000 max items

File: src/modelbench/run.py [hunk 2]
```diff
@@ -269,23 +269,25 @@ def calibrate(update: bool, file) -> None:
...
+        ModelGaugeSut.for_key("llama-3.1-8b-instruct-turbo-together"),
     ]
     if not ensure_ensemble_annotators_loaded():
         print("Can't load private annotators needed for calibration")
         exit(1)
 
-    benchmarks = [GeneralPurposeAiChatBenchmark()]
-    for l in Locale:
+    benchmarks = []
+    for l in [Locale.EN_US]:
```
Comment (bkorycki) [inline src/modelbench/run.py:280]: I might have missed this, but is 1.0 only going to include english prompts?

**Review by bkorycki — COMMENTED** [review]
Thanks for doing this!

File: src/modelbench/run.py (hunk 1 above)
Comment (wpietri) [reply src/modelbench/run.py:282]: Ah, sorry, I shouldn't have left that in. The calibration set should be all the items, but I was doing practice runs with smaller numbers. 2000 is just larger than the 1000 max. I'll change it so it's always everything.

File: src/modelbench/run.py (hunk 2 above)
Comment (wpietri) [reply src/modelbench/run.py:280]: Yup. @bollacker knows more, but there were a number of challenges, so new languages will come shortly after 1.0.

**Review by bkorycki — APPROVED** [review]

**Review by bollacker — APPROVED** [review]
I see nothing obviously wrong.

File: src/modelbench/run.py (hunk 2 above)
Comment (bkorycki) [reply src/modelbench/run.py:280]: Ah ok, sounds good

**Review by dhosterman — APPROVED** [review]
👍

**Pull request merged**